### ISSUE TITLE
Set aria-current attributes on next/link

### ISSUE THREAD

**Issue by alexandernanberg** [vercel/next.js#6014]
# Feature request

## Is your feature request related to a problem? Please describe.
Right now you have to do this yourself and I'd bet many forget this. It would be really nice to have this included in `next/link` by default.

## Describe the solution you'd like
Set a `aria-current` attribute on the active link(s). I could work on this, but I might need some guidance on how this should be implemented since we probably have to wrap the link component in a `withRoute` hoc.

## Describe alternatives you've considered
N/A

## Additional context
https://www.w3.org/TR/wai-aria-1.1/#aria-current
